docs/balances/FAQs.md

```diff
@@ -8,6 +8,50 @@ tags: [Balances]
 
 Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.
 
+Valuations are dated, and only transactions with a timestamp on or before a valuation's date will be reflected in the valuation.
+
+Balances are not dated and are continuously updated; therefore, they will reflect any transactions booked to the portfolio as they occur.
+
+### Example
+
+```mermaid
+gantt
+dateFormat YYYY-MM-DD HH:mm
+axisFormat %m-%d %H:%M
+title Dated valuations
+todayMarker off
+
+    section Valuation
+    Valuation 2024-02-19 0GBP :active, val1, 2024-02-19 17:00, 62h
+    Deposit 100GBP :milestone, deposit, 2024-02-20 02:00, 0h
+    Valuation 2024-02-20 calculated :milestone, calc1, 2024-02-20 06:00, 0h
+    Valuation 2024-02-20 100GBP :active, val1, 2024-02-20 06:00, 49h
+    Bonus awarded 10GBP :milestone, bon, 2024-02-20 12:00, 0h
+    Valuation 2024-02-21 calculated :milestone, calc2, 2024-02-21 06:00, 0h
+    Valuation 2024-02-21 110GBP :active, val2, 2024-02-21 06:00, 25h
+```
```

Comment:
I've been thinking about these charts as they didn't look right to me but I could put my finger on why, but I think I know why now: they are showing when a valuation is readable but showing transaction events alongside, which are taken into account for a valuation even if it is not readable.

I think maybe we remove them to unblock and have a think about how we can diagram them in way that can't be misinterpreted.